mkdocs.yml

```diff
@@ -120,7 +120,7 @@ nav:
           - Evaluations: docs/guides/evaluations.md
           - DORA metrics: docs/guides/dora.md
           - OpenTelemetry observability: docs/guides/otel.md
-          - Metadata: docs/guides/metadata.md
+          - Context metadata: docs/guides/metadata.md
```

Comment:
when changing this title here, please also change the title of the corresponding page accordingly

Reply:
The corresponding page is entitled "Context metadata" so I am just changing mkdocs to match that

Reply:
perfect